webview/src/util/props.ts

```diff
@@ -15,9 +15,6 @@ export const clickAndEnterProps: <T>(
   onKeyDown: React.KeyboardEventHandler<T>
 } = (handler, textsForSelection = [], checkForTooltip = false) => ({
   onClick: e => {
-    e.preventDefault()
-    e.stopPropagation()
```

Comment:
I could not find a reason for these being here. Does anyone recall the reason?

Reply:
I would move this inside the following `if` branch. We do want to stop any default or added behaviour if the user is selecting., but in a normal context, there is no reason for these to be there and it might cause some unexpected behaviour.